src/provider/camunda-platform/properties/FormTypeProps.js

```diff
@@ -35,43 +46,101 @@ export function FormTypeProps(props) {
 function FormType(props) {
   const { element } = props;
 
-  const modeling = useService('modeling');
   const translate = useService('translate');
-
+  const bpmnFactory = useService('bpmnFactory');
   const businessObject = getBusinessObject(element);
+  const commandStack = useService('commandStack');
+
+  let extensionElements = businessObject.get('extensionElements');
 
   const getValue = () => {
     if (isDefined(businessObject.get('camunda:formKey'))) {
       return 'formKey';
     } else if (isDefined(businessObject.get('camunda:formRef'))) {
       return 'formRef';
+
```

Comment:
```suggestion
```

Reply:
Or add an empty line after https://github.com/bpmn-io/bpmn-js-properties-panel/pull/574/files#diff-88ce5e1f5e98eb613c3b7855f9da6b44890685dbbafb3d46c25c3f69fcd83d38R58, but let's have it consistent